core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -251,3 +251,79 @@ message GetDomainInfoResponse {
   // Vrf contains the VRF public key.
   keyspb.PublicKey vrf = 3;
 }
+
+message BatchGetEntriesRequest {
```

Comment:
This is a backend / trusting API.  If the API responds with a non-error message, it succeeded.